hrm-domain/hrm-core/contact-job/contact-job-complete.ts

```diff
@@ -174,6 +177,21 @@ export const pollAndProcessCompletedContactJobs = async (jobMaxAttempts: number)
         const completedJob: CompletedContactJobBody = JSON.parse(m.Body);
 
         if (completedJob.attemptResult === ContactJobAttemptResult.SUCCESS) {
+          // I am not sure if I did this rught, but I am supposed to call the createContactJob and
+          // pass the additionalPayload
+          const contact = await getById(completedJob.accountSid, completedJob.contactId);
+          await createContactJob(completedJob)({
```

Comment:
```suggestion
          await createContactJob()({
```
The first parameter is for a transaction if this is being done as part of a DB transaction, we can leave it blank